editor/create_dialog.cpp

```diff
@@ -111,7 +111,7 @@ void CreateDialog::popup_create(bool p_dont_clear, bool p_replace_mode, const St
 
 	if (p_replace_mode) {
 		select_type(p_select_type);
-		set_title(vformat(TTR("Change %s Type"), base_type));
+		set_title(vformat(TTR("Convert %s from %s"), p_select_name, p_select_type));
```

Comment:
`CreateDialog` is used in visual script with `p_replace_mode` enabled but no other parameters parsed:

```
void VisualScriptEditor::_change_base_type() {
	select_base_type->popup_create(true, true);
}
```

This change would make the title look weird, and probably needs addressing. (Though visual script is on its way out so, maybe we don't care).

Reply:
The fix is easy to do, so no reason to break the editor IMO (it's still there).
Here's the relevant line:
https://github.com/godotengine/godot/blob/f2a61684143af02a8cbe7002645af817607f9bd6/modules/visual_script/editor/visual_script_editor.cpp#L2939
`p_select_name` can't be determined in VisualScript though.

Reply:
@KoBeWi What should I set `p_select_name` & `p_select_type` to in ` select_base_type->popup_create(true, true)`?

Reply:
`p_select_type` can be either `base_type` or `script->get_instance_base_type()` I think.
As for `p_select_name`... maybe something generic, like "base type" or "VisualScript"? You could also add a special case for empty `p_select_name`.

Reply:
Something generic would be hard to localize. I'd go for handling this case of an empty value properly in the dialog.

Reply:
```
if (!p_select_name.size()) {
			set_title(vformat(TTR("Convert %s from %s"), p_select_name, p_select_type));
		} else {
			set_title(vformat(TTR("Convert from %s type base script"), p_select_type));
		}
```
I think this will do well. Let me know your thoughts.

Reply:
Should be `p_select_name.is_empty()`.

Reply:
Done :)